### DISCUSSION TITLE
The first computer mouse: how many buttons did it have, and what material was it made of?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#204] [Galaxy Trivia] [answered]
**Hint:** 🖱🐁

**Comment by PavelLaptev** [comment, marked as answer]
One button, wood. Made in 1981, by Xerox PARC
![image](https://user-images.githubusercontent.com/18498712/222969702-155e3053-f95c-4852-948d-9e87fbb269bf.png)
![image](https://user-images.githubusercontent.com/18498712/222969769-b1ee0748-f70d-45d2-a659-1531acde28a8.png)
